### DISCUSSION TITLE
针对这次课件无法下载的问题讨论

### DISCUSSION THREAD

**Discussion by Dongni03** [52beijixing/smartedu-download#23] [General]
现在课件下载不了，大哥们是怎么搞的？

**Comment by Thams94** [comment]
我也遇到了。显示下载错误

**Comment by Gaukay** [comment]
出现错误，课件还是没法下载
![屏幕截图 2024-09-22 222956](https://github.com/user-attachments/assets/b64d0594-dba2-48ee-aa88-f248a9f5998f)

**Comment by leemuhe** [comment]
the same issue +1
